internal/pkg/skuba/addons/kured_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+func TestGetKuredImage(t *testing.T) {
```

Comment:
`TestGetKuredImage` vs `Test_renderContext_KuredImage` ...
Out of curiosity, any reason this one doesn't have underscore and the other one has?

Reply:
Generate test skeleton using gotests. I think it one is function "TestGetKuredImage" and other one is "Test_renderContext_KuredImage"

Reply:
Hmm, not sure if I am following. I am curious about the in consistency of your test function names. Any reason one involves underscore and the other one doesn't have underscore?

Reply:
Use the tool [gotests](https://github.com/cweill/gotests) to generate skeleton:

function: GetKuredImage => TestGetKuredImage
method: (renderContext renderContext) KuredImage => Test_renderContext_KuredImage

Reply:
Ah, got it. I've never used it to generate the skeleton before. Something learned. Thanks.